templates/boltzmann.js

```diff
@@ -1256,6 +1268,19 @@ function authenticateJWT ({
   algorithms=['RS256'],
   storeAs = 'user'
 } = {}) {
+  if (!Array.isArray(algorithms)) {
+    if (typeof algorithms == 'string') {
+      algorithms = [algorithms]
+    } else {
+      throw new TypeError('The `algorithms` config option for JWTs must be an array of strings')
+    }
+  }
```

Comment:
I might suggest `algorithms = [].concat(algorithms)` which is a no-op on arrays, but turns any single value into an array of that one value. Then we can say `algorithms.forEach(xs => assert(typeof xs == 'string'))` or similar?